src/components/SettingsPage/CleanButton.js

```
@@ -20,32 +21,35 @@ type Props = {
 
 type State = {
   opened: boolean,
+  fallbackOpened: boolean,
   isLoading: boolean,
 }
 
 class CleanButton extends PureComponent<Props, State> {
   state = {
     opened: false,
+    fallbackOpened: false,
     isLoading: false,
   }
 
   open = () => this.setState({ opened: true })
 
   close = () => this.setState({ opened: false })
+  closeFallback = () => this.setState({ fallbackOpened: false })
 
   action = async () => {
     if (this.state.isLoading) return
     try {
       this.setState({ isLoading: true })
       await softReset({ cleanAccountsCache: this.props.cleanAccountsCache })
-    } finally {
-      this.setState({ isLoading: false })
+    } catch (err) {
```

`logger.error` it. (we should never silent errors for logs)